assets/styles/components/_free-text.scss

```diff
@@ -0,0 +1,51 @@
+/*------------------------------------*\
+  #Footer
+\*------------------------------------*/
+
+.govuk-margin-free-text{
+
+    margin-bottom: 40%;
+}
+
+.govuk-border-right-free-text{
+    border-right-width: 5px;
+    border-color: $ccs-colour;
+    border-color: var(--sub-color);
+    border-right-style: solid;
+    
+}
+
+.wysiwyg-content-free-text{
+    font-family: Helvetica, Arial, sans-serif; font-size: 14px; line-height: 23px;
+    color: #0b0c0c;
+    margin-top: 100px;
+    margin-bottom: 50px;  
+}
+
+.font-free-text{
+    font-size: 30px;
+    font-weight: bold;
+}
+
+.wysiwyg-content-free-text-left-banner{
+    font-family: Helvetica, Arial, sans-serif; font-size: 20px; line-height: 15px;
+    color: #0b0c0c;
+    margin-top: 100px;
+}
+
+
+
+.wysiwyg-content-free-text-content-field{
+    font-family: Helvetica, Arial, sans-serif; font-size: 14px; line-height: 23px;
+    color: #0b0c0c;

```

Comment:
Just a general comment on how this could be refactored in future.

` .wysiwyg-content-free-text-content-field` and ` .wysiwyg-content-free-text` appear very similar ... I would be tempted to merge them and add new classes for specific differences. 

E.g. you could do something like:

```
/* New base class that we can use for generic default styling */
.wysiwyg-content-free-text-base {
    color: #0b0c0c;
    font-family: Helvetica, Arial, sans-serif; 
    font-size: 14px; 
    line-height: 23px;
}

/* specific overrides here */
.wysiwyg-content-free-text {
    margin-bottom: 50px;  
    margin-top: 100px;
}

.wysiwyg-content-free-text-left-banner {
    font-size: 20px; 
    line-height: 15px;
    margin-top: 100px;
}
```

This reduces repition and also reduces the amount of CSS code you have and removes duplication of code.

Then in the HTML you could combine these classes as necessary, e.g.

```
<div class="wysiwyg-content-free-text-base  wysiwyg-content-free-text"></div>

<div class="wysiwyg-content-free-text-base  wysiwyg-content-free-text-left-banner"></div>
etc.
```